### PR TITLE
Feature mobalize module migration

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**
Aligned with Skyux request to move our modules into the shared folder. updated package json to point to the new location of stache module.

### CODE REVIEW

File: src/app/app-extras.module.ts [hunk 1]
```diff
@@ -1,22 +1,18 @@
 import { NgModule } from '@angular/core';
```
Comment (Blackbaud-SteveBrush) [inline src/app/app-extras.module.ts]: Newline between namespaces, please.

File: src/app/shared/stache/modules/breadcrumbs/breadcrumbs.component.spec.ts [hunk 2]
```diff
@@ -53,13 +54,12 @@ describe('StacheBreadcrumbsComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [
-        StacheNavModule
-      ],
       declarations: [
+        StacheNavComponent,
```
Comment (Blackbaud-SteveBrush) [inline src/app/shared/stache/modules/breadcrumbs/breadcrumbs.component.spec.ts]: Why the change? Breadcrumbs doesn't own the `StacheNavComponent`...

File: src/app/shared/stache/modules/code-block/code-block.component.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ declare let Prism: any;
 import 'prismjs/prism';
 import 'prismjs/components/prism-typescript';
 import 'prismjs/plugins/normalize-whitespace/prism-normalize-whitespace';
+require('style-loader!prismjs/themes/prism.css');
```
Comment (Blackbaud-SteveBrush) [inline src/app/shared/stache/modules/code-block/code-block.component.ts]: What's the reasoning for adding it to every component, instead of the module, once?

File: src/app/shared/stache/modules/breadcrumbs/breadcrumbs.component.spec.ts (hunk 2 above)
Comment (Blackbaud-BrandonJones) [inline src/app/shared/stache/modules/breadcrumbs/breadcrumbs.component.spec.ts]: We can't use the NavModule in any of the code run by skyux, and it needs the component in order to run the test...

File: src/app/shared/stache/modules/sidebar/sidebar.component.spec.ts [hunk 4]
```diff
@@ -55,14 +56,13 @@ describe('StacheSidebarComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [
-        StacheNavModule
-      ],
       declarations: [
+        StacheNavComponent,
```
Comment (Blackbaud-SteveBrush) [inline src/app/shared/stache/modules/sidebar/sidebar.component.spec.ts]: Same as breadcrumbs, above...

File: src/app/shared/stache/modules/table-of-contents/table-of-contents.component.spec.ts [hunk 5]
```diff
@@ -18,15 +19,14 @@ describe('StacheTableOfContentsComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [
-        StacheNavModule
-      ],
       declarations: [
+        StacheNavComponent,
```
Comment (Blackbaud-SteveBrush) [inline src/app/shared/stache/modules/table-of-contents/table-of-contents.component.spec.ts]: Same as breadcrumbs, above.

File: src/app/shared/stache/modules/code-block/code-block.component.ts (hunk 3 above)
Comment (Blackbaud-BrandonJones) [inline src/app/shared/stache/modules/code-block/code-block.component.ts]: Skyux cannot import the module anywhere.  The modules in this case are specifically for our exported StacheModule.    The components that get pulled into the skyAppModule still need access to this, and it blows up if we require the module.

Comment (Blackbaud-SteveBrush) [inline src/app/shared/stache/modules/code-block/code-block.component.ts]: I tested this locally: place this `require` back in the code block module. Also include it above the `require('style-loader!./app.scss');` in the app-extras file. That way, we can preserve the location of the css file in the module, but also implement it for stache SPA. (This is how we're handling the smoothscroll polyfill as well.)
